src/components/FlowRunCreateForm.vue

```diff
@@ -149,6 +154,11 @@
   const name = ref<string>(generateRandomName())
   const parameters = ref<SchemaValues>(combinedParameters.value)
   const stateMessage = ref<string>('')
+  const workQueueName = ref<string | null>(props.deployment.workQueueName)
+  const workPoolName = ref<string | null>(props.deployment.workPoolName)
+
+  const { workPool } = useWorkPool(props.deployment.workPoolName ?? '')
+  const workQueueComboboxLabel = computed(() => `Work Queue for ${workPoolName.value} (Optional)`)
```

Comment:
Double checking - is `Work Queue` correct for the label here or should it be `Work queue`?

Reply:
I had to go back and check the style guide! I think labels should be using title case which means that all words expect for articles and short prepositions (which would include for) should be upper case. That also led me to add the capitalization of 'Optional'